packages/assertions/tm-assertions/tm-extend.json

```diff
@@ -0,0 +1,64 @@
+{
+    "title": "tm-extend",
+    "description": "A Thing Model MUST use at least one links entry with \"rel\":\"tm:extends\" that targets to a Thing Model that is be extended.",
+    "$schema ": "http://json-schema.org/draft/2019-09/schema#",
+    "is-complex":false,
+    "type": "object",
+    "properties": {
```

Comment:
`@type` validation should be just in case

Reply:
ok will do it